CHANGELOG.md

```diff
@@ -0,0 +1,427 @@
+# Changelog
+
+#### 2.35.1
+
+Release date: (July 12, 2017)
+
+-   Prevent NullPointerException when Parameterized Trigger starts
+    multiple builds and rejects some of them due to the missing
+    Job/Build permission
+    ([JENKINS-45471](https://issues.jenkins-ci.org/browse/JENKINS-45471))
+
+#### 2.35
+
+Release date: (July 10, 2017)
+
+-   [Fix security
+    issue](https://jenkins.io/security/advisory/2017-07-10/)
+
+#### 2.34
+
+Release date: (Jun 19, 2017)
+
+-   Update Jenkins core minimal requirement to 1.642.3
+-   Trigger config: Handle missing ancestor job when selecting
+    Downstream Projects in non-job
+    items ([JENKINS-32527](https://issues.jenkins-ci.org/browse/JENKINS-32527))
+
+#### 2.33
+
+Release date:  (Feb 26, 2017)
+
+-   Allow passing current build parameters to the projects being
+    triggered via Parameterized Post-Build Trigger ([PR
+    \#99](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/99))
+-   Allow setting downstream projects to be triggered from child
+    projects ([PR
+    \#106](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/106))
+-   Prevent error when triggering a list of project with a trailing
+    comma ([PR
+    \#111](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/111))
+-   Prevent NullPointerException in the case of race condition with
+    project enabling/disabling ([PR
+    \#110](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/110))
+-   Ensure that FileParameter file is closed after its reading when
+    constructing parameters ([PR
+    \#102](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/102))
+
+#### 2.32
+
+Release date: (Jul 26, 2016)
+
+-   Log a message instead of throwing a NullPointerException when not
+    being able to load a build's workspace ([PR
+    \#105](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/105))
+
+#### 2.31
+
+Release date: (Dec 16, 2015)
+
+-   Convert to use the new plugin parent pom
+    ([JENKINS-34474](https://issues.jenkins-ci.org/browse/JENKINS-34474))
+
+#### 2.30
+
+Release date: (Dec 16, 2015)
+
+-   Fix the sub project list visualization if a user has Item.DISCOVER
+    without Item.READ (JENKINS-31727)
+-   Minor fixes and improvements ([PR
+    \#91](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/91))
+
+#### 2.29
+
+Release date: (Sep 10, 2015)
+
+-   Workflow compatibility fixes
+-   Extra unit tests for JENKINS-30040
```

Comment:
the Jenkins references could be replaced by links

Reply:
Yes, just too much work for a historical changelog. It was as is on the Wiki